### PR TITLE
Add responded_by to PerConfirmation event details

### PR DESCRIPTION
**Pull request by Thource**
This is the API side of this work, there is still frontend work to do, but I'm releasing this so that I can complete the frontend work more easily.

### Jira link

P4-3575

### What?

I have added/removed/altered:

- [ ] Added `responded_by` field to PerConfirmation events.

### Why?

I am doing this because:

- It will allow the frontend to display the authors of PERs.

### CODE REVIEW

File: lib/tasks/populate_per_confirmation_with_responded_by.rake [hunk 1]
```diff
@@ -0,0 +1,18 @@
+GITHUB_FRAMEWORK_URI = 'https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks.git'.freeze
```
Comment (matt-crowson-moj) [inline lib/tasks/populate_per_confirmation_with_responded_by.rake:1]: For my benefit what are lines 1-3 doing?

Comment (Thource) [reply lib/tasks/populate_per_confirmation_with_responded_by.rake:1]: Copy pasta :D I'll remove them

File: app/controllers/api_controller.rb [hunk 2]
```diff
@@ -57,6 +57,8 @@ def info_for_paper_trail
 private
 
   def set_sentry_transaction_id
+    return unless Sentry.initialized?
```
Comment (matt-crowson-moj) [inline app/controllers/api_controller.rb:60]: Is this to resolve an issue with the initialisation of Sentry?

Comment (Thource) [reply app/controllers/api_controller.rb:60]: Yeah, this is unrelated to the PER author stuff, it just means that we can run the API locally without setting the sentry env vars now

**Pull request merged**